docs/machine-learning/tutorials/object-detection-model-builder.md

```diff
@@ -0,0 +1,261 @@
+
+> If you want to skip the data labeling steps below, you can [download this version of the dataset](https://aka.ms/mlnet-object-detection-tutorial-assets) and skip to [Create a console application](object-detection-model-builder.md#create-a-console-application).
+
+### Create a new VoTT project
+
+1. [Download the dataset](https://aka.ms/mlnet-object-detection-tutorial-dataset) of 50 stop sign images and unzip.
+1. [Download VoTT](https://github.com/Microsoft/VoTT/releases) (Visual Object Tagging Tool).
+1. Open VoTT and select **New Project**.
+
+![VoTT Home Screen](./media/object-detection-model-builder/vott.png)
+
+1. In **Project Settings**, change the **Display Name** to "StopSignObjDetection".
+1. Change the **Security Token** to *Generate New Security Token*.
+1. Next to **Source Connection**, select **Add Connection**.
+1. In **Connection Settings**, change the **Display Name** for the source connection to "StopSignImages", and select *Local File System* as the **Provider**. For the **Folder Path**, select the *Stop-Signs* folder which contains the 50 training images, and then select **Save Connection**.
+
+![VoTT New Connection Dialog](./media/object-detection-model-builder/vott-new-connection.png)
+
+1. In **Project Settings**, change the **Source Connection** to *StopSignImages* (the connection you just created).
+1. Change the **Target Connection** to *StopSignImages* as well. Your **Project Settings** should now look similar to this screenshot:
+
+![VoTT Project Settings Dialog](./media/object-detection-model-builder/vott-new-project.png)
+
+1. Select **Save Project**.
+
+### Add tag and label images
+
+You should now see a window with preview images of all the training images on the left, a preview of the selected image in the middle, and a **Tags** column on the right. This screen is the **Tags editor**.
+
+1. Select the first (plus-shaped) icon in the **Tags** toolbar to add a new tag.
+
+![VoTT New Tag Icon](./media/object-detection-model-builder/vott-new-tag-icon.png)
+
+1. Name the tag "Stop-Sign" and hit <kbd>Enter</kbd> on your keyboard.
+![VoTT New Tag](./media/object-detection-model-builder/vott-new-tag.png)
+1. Click and drag to draw a rectangle around each stop sign in the image. If the cursor does not let you draw a rectangle, try selecting the **Draw Rectangle** tool from the toolbar on the top, or use the keyboard shortcut <kbd>R</kbd>.
+1. After drawing your rectangle, select the **Stop-Sign** tag that you created in the previous steps to add the tag to the bounding box.
+1. Click on the preview image for the next image in the dataset and repeat this process.
+1. Continue steps 3-4 for every stop sign in every image.
+
+![VoTT Annotating Images gif](./media/object-detection-model-builder/vott-annotating.gif)
```

Comment:
Not sure if "gif" should be included in the alt text.

Reply:
Fixed with commit https://github.com/dotnet/docs/pull/23300/commits/91f513edf8be0075a44936913fd12eee66aa4904